source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -100,8 +100,7 @@ bool AMPseudoMotorControl::validSetpoint(double value) const
 
 bool AMPseudoMotorControl::addChildControl(AMControl *control)
 {
-	if(AMControl::addChildControl(control)){
-		connect( control, SIGNAL(connected(bool)), this, SLOT(updateStates()) );
```

Comment:
`AMPseudoMotorControl` is a subclass of `AMConnectedControl`. Here I changed to use `AMConnectedControl::addChildChontrol(control)`, which should be a proper usage and also did the connected() signal/slot connection